qemu/tests/uefi_check_boot_option.py

```diff
@@ -0,0 +1,61 @@
+import re
+import time
+import logging
+
+from avocado.utils import process
+
+from virttest import utils_misc
+from virttest import env_process
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Verify UEFI config setting in the GUI screen:
+    1) Boot up a guest.
+    2) Check splash-time in OVMF.log
+    3) If boot_info not None, check boot entry info
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def check_log_info(check_info_cmd, expected_result):
+        """
+        info check
+        """
+        output = process.system_output(check_info_cmd)
+        result = re.search(expected_result, str(output))
+        return result
+
+    check_info_cmd = params["check_info_cmd"]
+    expected_result = params["expected_result"]
+    if params.get("ovmf_log"):
+        error_context.context("Append debugcon parameter to "
+                              "qemu command lines.", logging.info)
+        ovmf_log = utils_misc.get_path(test.debugdir, params["ovmf_log"])
+        params["extra_params"] %= ovmf_log
+        params["start_vm"] = "yes"
+        env_process.process(test, params, env,
+                            env_process.preprocess_image,
+                            env_process.preprocess_vm)
+    check_info_cmd %= ovmf_log
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    time.sleep(60)
```

Comment:
suggest you use utils_misc.wait_for or add the value to cfg file.

Reply:
OK,I will rewrite those code,many thanks!